src/main/java/io/jenkins/plugins/designlibrary/HeteroList.java

```diff
@@ -43,7 +44,7 @@
 import org.kohsuke.stapler.StaplerRequest;
 
 @Extension
-public final class HeteroList extends UISample {
+public class HeteroList extends UISample {
```

Comment:
Rename to RepeatableList to make the class easier to find?

Reply:
Good idea, done + simplified the class :)